### PR TITLE
Remove Lifeline from spm_unit_broadband_subsidy

### LINKED ISSUE #5004: Remove Lifeline from spm_unit_broadband_subsidy
The [SPM technical documentation](https://www2.census.gov/programs-surveys/supplemental-poverty-measure/technical-documentation/spm_techdoc.pdf#page=16) doesn't mention Lifeline, suggesting it's not counted.

### PR DESCRIPTION
**Pull request by srilareddy**
Fixes #5004

### CODE REVIEW

File: policyengine_us/tests/policy/baseline/household/income/spm_unit/spm_unit_broadband_subsidy.yaml [hunk 1]
```diff
@@ -13,4 +13,22 @@
...
+    spm_unit_broadband_subsidy: 3
+
+- name: Otherwise sum ACP 0, EBB 3
+  period: 2023
+  input:
+    gov.simulation.reported_broadband_subsidy: false
+    acp: 1
+    ebb: 1
+  output:
+    spm_unit_broadband_subsidy: 2
```
Comment (PavelMakarchuk) [inline policyengine_us/tests/policy/baseline/household/income/spm_unit/spm_unit_broadband_subsidy.yaml:34]:
```suggestion
```

Dont need to add another test - one baseline unit test is fine - there is no significant other edge case to test

File: policyengine_us/tests/policy/baseline/household/income/spm_unit/spm_unit_broadband_subsidy.yaml [hunk 2]
```diff
@@ -13,4 +13,22 @@
     acp: 2
     ebb: 3
   output:
-    spm_unit_broadband_subsidy: 6
+    spm_unit_broadband_subsidy: 5
+
+- name: Otherwise sum ACP 1, EBB 1
```
Comment (PavelMakarchuk) [inline policyengine_us/tests/policy/baseline/household/income/spm_unit/spm_unit_broadband_subsidy.yaml:18]:
```suggestion
- name: Baseline subsidy test with relevant inputs
```

File: changelog_entry.yaml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+ - bump: minor
+   changes:
+    fixed: 
+      -Remove lifeline varibale from spm_unit_broadband_subsidy.
```
Comment (PavelMakarchuk) [inline changelog_entry.yaml:4]:
```suggestion
      - Remove lifeline variable from spm_unit_broadband_subsidy.
```

File: changelog_entry.yaml [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+      -Modified tests for spm_unit_broadband_subsidy.
```
Comment (PavelMakarchuk) [inline changelog_entry.yaml:5]:
```suggestion
      - Modified tests for spm_unit_broadband_subsidy.
```

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

**Review by PavelMakarchuk — APPROVED** [review]

**Review by MaxGhenis — APPROVED** [review]
